app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -114,25 +113,14 @@ private void listenToModelChanges() {
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
             .map(DatabaseChangeListener.ModelChange::getModel)
-            .subscribe(this::processUpdatedAttendee, Throwable::printStackTrace);
```

Comment:
Please take a look at attendees view - presenter again. I think there is no need to save attendees list with the presenter currently.
the presenter's list is not used anywhere other than binding into the layout. which is not used for any purpose currently.
The attendees' list for the adapter is directly supplied from the `loadAttendees` result.